examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/main.ios.kt

```diff
@@ -21,14 +22,29 @@ fun ComposeEntryPoint(): UIViewController =
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
             Text("How to use UIKitView inside Compose")
-            UIKitView(
-                factory = { MKMapView() },
-                modifier = Modifier.size(300.dp).border(2.dp, Color.Blue),
-                update = {},
-            )
+
+            BlueBorder {
+                UIKitView(
+                    factory = { MKMapView() },
+                    modifier = Modifier.size(300.dp),
+                    update = {},
+                )
+            }
 
             Spacer(Modifier.size(20.dp))
 
-            UseUITextField()
+            BlueBorder {
+                UseUITextField()
+            }
         }
     }
+
+/**
+ * Border to easily understand the size of UIKitView
+ */
+@Composable
+fun BlueBorder(content: @Composable () -> Unit) {
```

Comment:
The Jetpack Compose recomendation is to use modifier for this case:
```
    ComposeUIViewController {
        Column(
            Modifier
                .fillMaxSize()
                .windowInsetsPadding(WindowInsets.systemBars),
            horizontalAlignment = Alignment.CenterHorizontally
        ) {
            Text("How to use UIKitView inside Compose")

            UIKitView(
                factory = { MKMapView() },
                modifier = Modifier.padding(4.dp).border(2.dp, Color.Blue).size(300.dp),
                update = {},
            )

            Spacer(Modifier.size(20.dp))

            UseUITextField(Modifier.padding(4.dp).border(2.dp, Color.Blue))
        }
    }
```

Reply:
Done